Add UseBlacklist and UseBlocklist to runtime/syntax/sshdconfig.vim

samy-mahmoudi opened the pull request:
Add UseBlacklist to the sshdconfigKeyword syntax
group since support for blacklistd has been added
to OpenSSH since FreeBSD 11.1 and NetBSD 7.

https://www.freebsd.org/releases/11.1R/relnotes.html
http://www.netbsd.org/releases/formal-7/NetBSD-7.0.html

Problem:    UseBlacklist is not highlighted when editing sshd_config.
Solution:   Add UseBlacklist to sshdconfig.vim. (Samy Mahmoudi)

**Update (2020-10-07): add a second commit to this PR**
Add UseBlocklist to the sshdconfigKeyword syntax group
since blacklistd was renamed blocklistd upstream (NetBSD).
These changes have not been imported into all the relevant
operating systems (e.g. FreeBSD) yet, so maintain UseBlacklist
in this syntax group until the transition is done.

http://mail-index.netbsd.org/current-users/2020/06/15/msg038868.html
https://reviews.freebsd.org/rS363657

Problem:    UseBlocklist is not highlighted when editing sshd_config.
Solution:   Add UseBlocklist to sshdconfig.vim. (Samy Mahmoudi)

samy-mahmoudi committed:
Add UseBlacklist to runtime/syntax/sshdconfig.vim

Add UseBlacklist to the sshdconfigKeyword syntax
group since support for blacklistd has been added
to OpenSSH since FreeBSD 11.1 and NetBSD 7.

https://www.freebsd.org/releases/11.1R/relnotes.html
http://www.netbsd.org/releases/formal-7/NetBSD-7.0.html

Problem:    UseBlacklist is not highlighted when editing sshd_config.
Solution:   Add UseBlacklist to sshdconfig.vim. (Samy Mahmoudi)

brammool commented:
> Add UseBlacklist to the sshdconfigKeyword syntax
> group since support for blacklistd has been added
> to OpenSSH since FreeBSD 11.1 and NetBSD 7.
> 
> https://www.freebsd.org/releases/11.1R/relnotes.html
> http://www.netbsd.org/releases/formal-7/NetBSD-7.0.html
> 
> Problem:    UseBlacklist is not highlighted when editing sshd_config.
> Solution:   Add UseBlacklist to sshdconfig.vim. (Samy Mahmoudi)

It's up to the maintainer to handle this: Dominik Fischer.

-- 
FATHER:    Did you kill all those guards?
LAUNCELOT: Yes ...  I'm very sorry ...
FATHER:    They cost fifty pounds each!
                 "Monty Python and the Holy Grail" PYTHON (MONTY) PICTURES LTD

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///

samy-mahmoudi commented:
I have written to Dominik Fischer to ask for his help.

Besides, following the first link leads to [https://svnweb.freebsd.org/base?view=revision&revision=305476](https://svnweb.freebsd.org/base?view=revision&revision=305476) which may quicken understanding:

Added UseBlacklist option to sshd, which defaults to off.
To enable the functionality, use '-o UseBlacklist=yes' on
the command line, or uncomment in the sshd_config file.

dffischer commented:
I am sorry that this went under my radar and will care for it shortly. Thanks @samy-mahmoudi for notifying me again.

samy-mahmoudi commented:
Nice to see you again Dominik! You are welcome.

brammool commented:
Related to #5753

samy-mahmoudi committed:
Add UseBlocklist to runtime/syntax/sshdconfig.vim

Add UseBlocklist to the sshdconfigKeyword syntax group
since blacklistd was renamed blocklistd upstream (NetBSD).
These changes have not been imported into all the relevant
operating systems (e.g. FreeBSD) yet, so maintain UseBlacklist
in this syntax group until the transition is done.

http://mail-index.netbsd.org/current-users/2020/06/15/msg038868.html
https://reviews.freebsd.org/rS363657

Problem:    UseBlocklist is not highlighted when editing sshd_config.
Solution:   Add UseBlocklist to sshdconfig.vim. (Samy Mahmoudi)
